meta-balena-common/recipes-containers/docker-disk/docker-disk.bb

```
@@ -54,37 +62,78 @@ do_compile () {
 	_image_name="docker-disk-$RANDOM"
 	_container_name="docker-disk-$RANDOM"
 	$DOCKER rmi -f ${_image_name} > /dev/null 2>&1 || true
-	$DOCKER build -t ${_image_name} -f ${WORKDIR}/Dockerfile ${WORKDIR}
+	$DOCKER build ${no_cache} -t ${_image_name} -f ${WORKDIR}/Dockerfile ${WORKDIR}
 	$DOCKER run --privileged --rm \
 		-e BALENA_STORAGE=${BALENA_STORAGE} \
 		-e USER_ID=$(id -u) -e USER_GID=$(id -u) \
 		-e SUPERVISOR_FLEET="${SUPERVISOR_FLEET}" \
 		-e SUPERVISOR_VERSION="${SUPERVISOR_VERSION}" \
 		-e HELLO_REPOSITORY="${HELLO_REPOSITORY}" \
-		-e HOSTEXT_IMAGES="${HOSTEXT_IMAGES}" \
 		-e HOSTAPP_PLATFORM="${HOSTAPP_PLATFORM}" \
 		-e BALENA_API_ENV="${BALENA_API_ENV}" \
 		-e BALENA_API_TOKEN="${_token}" \
 		-e PARTITION_SIZE="${PARTITION_SIZE}" \
 		-e FS_BLOCK_SIZE="${FS_BLOCK_SIZE}" \
+		-e HOSTOS_APPS="${HOSTOS_APPS}" \
+		-e HOSTOS_VERSION="${HOSTOS_VERSION}" \
+		-e BALENA_ADMIN="${BALENA_ADMIN}" \
 		-v /sys/fs/cgroup:/sys/fs/cgroup:ro -v ${B}:/build \
 		--name ${_container_name} ${_image_name}
 	$DOCKER rmi -f ${_image_name}
 }
 
+python __anonymous() {
+    import urllib.request
+    import json
+    import hashlib
+
+    token = d.getVar('BALENAOS_TOKEN')
+    apiEnv = d.getVar('BALENA_API_ENV')
+    version = d.getVar('HOSTOS_VERSION')
+    semver = version.split('+')[0]
+    revision = version.split('+')[1][3:]
+    translation = "v6"
+    header = {}
+    if token is not None:
+        header = {'Authorization': "bearer " + token}
+
+    cksums = []
+    apps = d.getVar('HOSTOS_APPS')
+    for appName in apps.split():
+        url = 'https://api.%s/%s/application?$filter='  \
+            'slug%%20eq%%20%%27balena_os/%s%%27' % (apiEnv, translation, appName)
```

You should be able to do this as
```suggestion
        url = 'https://api.%s/%s/application(slug=%%27balena_os/%s%%27)' % (apiEnv, translation, appName)
```
I also don't think you need to escape the `'` as `%27`, but I'm assuming you're doing it in part to avoid escaping the `'` within the `'` delineated string